tst/MyInfoConnector.Console/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Singapore Gov MyInfoConnector")]
+[assembly: AssemblyDescription("SingPass integration library")]
```

Comment:
Naming here still follows the previous one.
```suggestion
[assembly: AssemblyTitle("MyInfoConnector")]
[assembly: AssemblyDescription(".Net Connector for MyInfo")]
```